terraform/projects/app-ecs-services/config/alerts/data-gov-uk.yml

```diff
@@ -32,3 +32,11 @@ groups:
     annotations:
         summary: "Index size of Elasticsearch for {{ $labels.job }} has decreased significantly"
         description: "The index size of Elasticsearch for {{ $labels.job }} has decreased by more than 100 documents in the last 30 minutes"
+  - alert: DataGovUk_HighSidekiqEnqueuedJobs
+    expr: sidekiq_enqueued_jobs > 1000
```

Comment:
The description says 100, but it looks like the query specifies 1000

Reply:
👍 thanks @brucebolt  - fixed in 13f6c5b